src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  display: block;
+  margin: 16px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-title {
```

Comment:
https://codeguide.academy/html-css.html#css-order 
it will be better to add empty line between different group of values